src/EulerHS/KVConnector/Flow.hs

```diff
@@ -747,7 +749,8 @@ findOneFromRedis meshCfg whereClause = do
   eitherKeyRes <- mapM (getPrimaryKeyFromFieldsAndValues modelName meshCfg keyHashMap) andCombinationsFiltered
   case foldEither eitherKeyRes of
     Right keyRes -> do
-      allRowsRes <- foldEither <$> mapM (getDataFromPKeysRedis meshCfg) (mkUniq keyRes)
+      latencyLogging <- liftIO $ fromMaybe False . (>>= readMaybe) <$> SE.lookupEnv "logRedisLatency"
```

Comment:
it's general pattern to make these ENV's Capital and separated by `_` 

`"LOG_REDIS_LATENCY"` also it would be good to attach some prefix specific to Euler so it won't conflict.